sw/device/tests/sim_dv/keymgr_sideload_kmac_test.c

```diff
@@ -93,8 +92,10 @@ bool test_main(void) {
   dif_kmac_t kmac;
   keymgr_testutils_startup(&keymgr, &kmac);
 
-  // Test KMAC sideloading.
-  test_kmac_with_sideloaded_key(&keymgr, &kmac);
+  // Identity generation is not really necessary for this test, but this
+  // call helps us reuse the sequence from keymgr-key-derivation base vseq
+  keymgr_testutils_generate_identity(&keymgr);
+  LOG_INFO("Keymgr generated identity at CreatorRootKey State");
```

Comment:
Thanks for fixing this.
Perhaps move this part to `keymgr_testutils_startup`, otherwise, we need to add it for the aes sideload test as well.